strax/storage/mongo.py

```diff
@@ -85,10 +85,10 @@ def _read_chunk(self, backend_key, chunk_info, dtype, compressor):
     def _saver(self, key, metadata, **kwargs):
         """See strax.Backend"""
         # Use the key to make a collection otherwise, use the backend-key
-        col = self.db[self.col_name if self.col_name is not None else str(key)]
```

Comment:
This option did not work anyway and is just confusing

Reply:
So if this didnt work im assuming that means there is no existing code this can break?

Reply:
No indeed, I actually tried to write a test for it but this old logic. But apart from this one line has abandoned this many collection option. See e.g. this line that is assuming there is only one collection:
https://github.com/AxFoundation/strax/blob/22452f2c3f93ae309115f30ce24e50d04c8c8b88/strax/storage/mongo.py#L193

So yeah, I guess it's impossible to break a formerly unused and broken option.